tests/amp-tag-and-attribute-sanitizer-private-methods-tests.php

```diff
@@ -246,136 +246,6 @@ public function get_attr_spec_rule_data() {
 				),
 				'expected' => AMP_Rule_Spec::NOT_APPLICABLE,
 			),
-
```

Comment:
The allowed protocol values are removed from the spec file, so this removes the tests for them.